doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
+-  Cordex GERICS REMO2015  lon differences above 10e-4 (:pull:`2334`) by :user:`mwjury`
+-  Download ICON grid without locking (:pull:`2359`) by :user:`bouweandela`
+-  Added ICON fixes for hfls and hfss (:pull:`2360`) by :user:`diegokam`
+-  Added ICON fix for rtnt (:pull:`2366`) by :user:`diegokam`
+-  Expanded ICON extra facets (:pull:`2379`) by :user:`schlunma`
+-  Add 10m height coordinate to SfcWind GFDL-CM4 instead of 2m height (:pull:`2385`) by :user:`Karen-A-Garcia`
+-  Cordex wrf381p: fix tas,tasmax,tasmin height (:pull:`2333`) by :user:`mwjury`
+-  Several minor fixes needed for marine BGC data. (:pull:`2110`) by :user:`ledm`
+
+Installation
+~~~~~~~~~~~~
+
+-  Updated iris pin to `iris>=3.6.1` (:pull:`2286`) by :user:`schlunma`
+-  Pin pandas yet again avoid new `2.2.1` as well (:pull:`2353`) by :user:`valeriupredoi`
+-  Pin esmpy <8.6.0 (:pull:`2402`) by :user:`valeriupredoi`
+-  Update Iris pin to avoid using versions with memory issues (:pull:`2408`) by :user:`chrisbillowsMO`
+
+Iris
+~~~~
+
+-  Handle warnings about invalid units for iris>=3.8 (:pull:`2378`) by :user:`schlunma`
+-  Remove custom fix for concatenation of aux factories now that bug in iris is solved (:pull:`2392`) by :user:`schlunma`
+-  Use `iris.FUTURE.save_split_attrs = True` to remove iris warning (:pull:`2398`) by :user:`schlunma`
+-  Ignored iris warnings about global attributes (:pull:`2400`) by :user:`schlunma`
+
+Preprocessor
+~~~~~~~~~~~~
+
+-  New preprocessor: `local_solar_time` (:pull:`2258`) by :user:`schlunma`
+-  Read derived variables from other MIP tables (:pull:`2256`) by :user:`bouweandela`
+-  Added special unit conversion m -> DU for total column ozone (toz) (:pull:`2270`) by :user:`schlunma`
+-  Allow cubes as input for `bias` preprocessor (:pull:`2183`) by :user:`schlunma`
+-  Add normalization with statistics to many statistics preprocessors (:pull:`2189`) by :user:`schlunma`
+-  More lazy `extract_levels` preprocessor function (:pull:`2120`) by :user:`bouweandela`
+-  Adding sfcWind derivation from uas and vas  (:pull:`2242`) by :user:`malininae`
+-  Update interval check in resample_hours (:pull:`2362`) by :user:`axel-lauer`
+-  Broadcast properly `cell_measures` when using `extract_shape` with `decomposed: True` (:pull:`2348`) by :user:`sloosvel`
+-  Compute volume from cell_area if available (:pull:`2318`) by :user:`enekomartinmartinez`
+-  Do not expand wildcards for datasets of derived variables where not all input variables are available (:pull:`2374`) by :user:`schlunma`
+-  Cache regridding weights if possible (:pull:`2344`) by :user:`schlunma`
+-  Modernize `regrid_time` and allow setting a common calendar for decadal, yearly, and monthly data (:pull:`2311`) by :user:`schlunma`
+-  Added unstructured linear regridding (:pull:`2350`) by :user:`schlunma`
+-  Add preprocessors `distance_metrics` and `histogram` (:pull:`2299`) by :user:`schlunma`
+
+Release
```

Comment:
I would remove this section and move the pull requests to other sections, as the title Release does not convey much information for readers of the changelog.

Reply:
Removed here: fc86baf